package.json

```diff
@@ -23,6 +24,13 @@
   "author": "Daniel Lytkin <dan.lytkin@gmail.com>",
   "license": "MIT",
   "devDependencies": {
-    "typescript": "^2.2.1"
+    "@babel/core": "^7.24.7",
+    "@babel/preset-env": "^7.24.7",
+    "@babel/preset-typescript": "^7.24.7",
+    "@types/jest": "^29.5.12",
+    "babel-jest": "^29.7.0",
+    "jest": "^29.7.0",
+    "type-assertions": "^1.1.0",
+    "typescript": "^5.5.3"
```

Comment:
I had to bump the `typescript` version as older versions were having problems with some of the types from `jest` & `babel`. I tried using `"skipLibCheck": true` but this didn't help. It only started working from TypeScript version `4.x` and up so I just went with the current latest. I also double checked the generated output and it still seems on point (i.e. not using any new features beyond ES5).